src/main/java/com/answerdigital/academy/answerking/model/Category.java

```diff
@@ -68,6 +71,7 @@ public void removeProduct(final Product product) {
     public Category(final String name, final String description) {
```

Comment:
Could we please move this constructor further up, above addProduct & removeProduct?